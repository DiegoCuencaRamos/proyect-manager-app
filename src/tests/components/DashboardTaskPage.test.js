import React from 'react'
import { shallow } from 'enzyme'
import DashboardTasktPage from '../../components/DashboardTaskPage'

let wrapper

beforeEach(() => {
    wrapper = shallow(<DashboardTasktPage />)
})

test('Should render DashboardTaskPage correctly', () => {
    expect(wrapper).toMatchSnapshot()
}) 

test('Should setup default luyout state', () => {
    expect(wrapper.find('ItemsLayout').prop('layout')).toBe('')
})

test('Should setup luyout state', () => {
    const layoutData = 'calendar'
    wrapper.find('LayoutSelectors').prop('handleLayoutChange')(layoutData)
    expect(wrapper.find('ItemsLayout').prop('layout')).toBe(layoutData)
})