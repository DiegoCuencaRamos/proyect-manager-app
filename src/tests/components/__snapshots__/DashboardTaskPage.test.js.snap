// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render DashboardTaskPage correctly 1`] = `
<section>
  <Title
    description="From here you will be able to visualize your task with in the chosen proyect"
    isDashboard={true}
    title="Task Dashboard"
  />
  <LayoutSelectors
    handleLayoutChange={[Function]}
  />
  <ItemsLayout
    layout=""
  />
</section>
`;
